Quality metric update (#318)
biometricITC/cPP-biometrics: Protection Profile/BiocPP.adoc

===== FIA_MBV_EXT.2 Quality of biometric samples for biometric verification [[FIA_MBV_EXT.2]]

> *FIA_MBV_EXT.2.1* The TSF shall only use biometric samples of sufficient quality for verification. As such, sample data shall have [*selection*: _no independent quality metric, [*assignment*: quality metric standard]_].

Reviewer: * independent -> internal
 * add threshold

```suggestion
*FIA_MBV_EXT.2.1* The TSF shall only use biometric samples of sufficient quality for verification. As such, sample data shall have [*selection*: _an internal quality metric_, [*assignment*: _quality metric standard_]], using a threshold of [*assignment*: _quality metric threshold_].
```

> *Application Note {counter:remark_count}*:: ST author may specify quality standards if the TOE follows such standard, or provide a description of the metrics used.

Reviewer: This application note can be removed (was removed for enrolment quality).
```suggestion
```
